### DISCUSSION TITLE
'FunctionSeries' is not a registered object

### DISCUSSION THREAD

**Discussion by WayneX-Wong** [idaholab/moose#27704] [Q&A Modules: General] [answered]
Hi,

I try to use functional expansion tools, so I set `FUNCTIONAL_EXPANSION_TOOLS  := yes`

but it gives `A 'FunctionSeries' is not a registered object` error.

I then clean with `git clean -xfd` `make clobberall` and remake, but still get this error

**Comment by GiudGiud** [comment, marked as answer]
Hello

This is a known past issue

It s already fixed in a recent moose version
